GariunaiCloud-ToolSharing/PresentationLayer/ListingsController.cs

```diff
@@ -155,5 +155,17 @@ public async Task<IActionResult> DeleteListing(long id)
             await _listingService.DeleteListingAsync(id);
             return Ok();
         }
+        /// <summary>
+        /// Search for listings that match some string
+        /// </summary>
+        /// <param name="searchString"></param>
+        /// <returns>Return listings that contain the searchString in their titles</returns>
+        [HttpGet("{searchString}")]
+        public async Task<IActionResult> SearchListings(string searchString)
```

Comment:
To make this more REST'full add this as query param not route param:
* HttpGet("{searchString}") -> HttpGet()
* add [FromQuery] annotation to searchString